lib/matplotlib/animation.py

```diff
@@ -1079,7 +1079,11 @@ def func(current_frame: int, total_frames: int) -> Any
 
         def _pre_composite_to_white(color):
             r, g, b, a = mcolors.to_rgba(color)
-            return a * np.array([r, g, b]) + 1 - a
+            if type(writer) == PillowWriter:
```

Comment:
this needs to be addressed

```suggestion
            if isinstance(writer, PillowWriter):
```